app/javascript/components/rooms/RoomPlaceHolder.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, Placeholder } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCopy } from '@fortawesome/free-regular-svg-icons';
+import { faChalkboardUser } from '@fortawesome/free-solid-svg-icons';
+
+export default function RoomPlaceHolder() {
+  return (
+    <Card className="rooms-placeholder" style={{ height: '19rem', overflow: 'hidden' }} border="light">
+      <Card.Body style={{ maxHeight: '12rem', overflow: 'hidden' }} className="room-placeholder-top pb-0">
+        <FontAwesomeIcon icon={faChalkboardUser} size="3x" className="mb-4" />
+        <Placeholder as={Card.Title} animation="glow">
+          <Placeholder xs={12} size="sm" />
+        </Placeholder>
+        <Placeholder as={Card.Text} animation="glow">
+          <Placeholder xs={7} size="xs" /> <Placeholder xs={4} size="xs" /> <Placeholder xs={3} size="xs" />
+        </Placeholder>
+      </Card.Body>
+      <Card.Body style={{ maxHeight: '7rem', overflow: 'hidden' }} className="pt-0">
+        <Placeholder as={Card.Text} animation="glow">
+          <Placeholder xs={3} size="xs" /> <Placeholder xs={2} size="xs" />
```

Comment:
Is placeholder required here? Is there a way we can do this across all fields in the app without manually specifying each one?

Reply:
I don't think I get  exactly what you mean?

Reply:
Alright, but what should I set in place of the actual room card?
I'm doing an optimistic update and the `RoomPlaceHolder` component was made to replace the actual room until it gets created?
As for making a generic placeholder that fits all components, I don't know how we can do that.
When working on this, the cleanest way that came to my mind is the fallowing where each component will have its mock.